src/components/layout/Quiz/StateQuestion.js

```diff
@@ -0,0 +1,119 @@
+import React from "react"
+import { connect } from "react-redux"
+import PropTypes from "prop-types"
+
+// Redux
+import { setLocation } from "../../../actions/quizActions"
+
+// Material UI
+import {
+  FormControl,
+  InputLabel,
+  makeStyles,
+  MenuItem,
+  Select,
+  Typography,
+} from "@material-ui/core"
+
+const STATES = [
+  "AL",
+  "AK",
+  "AZ",
+  "AR",
+  "CA",
+  "CO",
+  "CT",
+  "DE",
+  "FL",
+  "GA",
+  "HI",
+  "ID",
+  "IL",
+  "IN",
+  "IA",
+  "KS",
+  "KY",
+  "LA",
+  "ME",
+  "MD",
+  "MA",
+  "MI",
+  "MN",
+  "MS",
+  "MO",
+  "MT",
+  "NE",
+  "NV",
+  "NH",
+  "NJ",
+  "NM",
+  "NY",
+  "NM",
+  "NY",
+  "NC",
+  "ND",
+  "OH",
+  "OK",
+  "OR",
+  "PA",
+  "RI",
+  "SC",
+  "SD",
+  "TN",
+  "TX",
+  "UT",
+  "VT",
+  "VA",
+  "WA",
+  "WV",
+  "WI",
+  "WY",
+]
+
+const useStyles = makeStyles(theme => ({
+  formControl: {
+    minWidth: 200,
+    marginTop: 10,
+    marginLeft: 20,
```

Comment:
I recommend you remove the `marginLeft`. To my eyes, the layout looks better when the question is aligned with the header.